python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1757,10 +1763,77 @@ def add_one_ufunc(a):
 def test_fallback_raises_error(monkeypatch):
     with monkeypatch.context() as monkeycontext:
         monkeycontext.setenv("CUDF_PANDAS_FAIL_ON_FALLBACK", "True")
-        with pytest.raises(ProxyFallbackError):
+        with pytest.raises(FallbackError):
             pd.Series(range(2)).astype(object)
 
 
+def mock_mean_memory_error(self, *args, **kwargs):
+    raise MemoryError()
+
+
+def mock_mean_rmm_error(self, *args, **kwargs):
+    raise RMMError(1, "error")
+
+
+def mock_mean_not_impl_error(self, *args, **kwargs):
+    raise NotImplementedError()
+
+
+def mock_mean_attr_error(self, *args, **kwargs):
+    raise AttributeError()
+
+
+def mock_mean_type_error(self, *args, **kwargs):
+    raise TypeError()
+
+
+@pytest.mark.parametrize(
+    "mock_mean, err, match_str",
+    [
+        (
+            mock_mean_memory_error,
```

Comment:
Since these are all single-use functions I'd suggest using lambdas instead.

Reply:
I found this [stackoverflow](https://stackoverflow.com/questions/8294618/define-a-lambda-expression-that-raises-an-exception) on raising exceptions in lambda functions. Most of them are hard to understand except maybe this one. WDYT?
`lambda x: exec('raise(Exception(x))')`

Reply:
Yikes I'm sorry, I thought I responded to this. I totally forgot about the lambda exceptions restriction. No, we shouldn't introduce `exec` calls just for this, the current code is fine.